MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -25,7 +25,11 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
     : _mesh(mesh), _point(point)
 {
     auto const node_ids = mshNodeSearcher.getMeshNodeIDs(_point);
-    assert(node_ids.size() == 1);
+    if (node_ids.size() != 1)
```

Comment:
Can `node_ids.size()` be bigger than one? E.g. if the search length is too big. If so, I suggest that there be two distinct cases and two different messages:
`if (node_ids.empty()) ... not found ...`
and
`if (node_ids.size() > 1) ... maybe search length too big ...`